.github/workflows/ci-mobile-save-for-later-user-deletion.yaml

```diff
@@ -0,0 +1,35 @@
+name: CI
+on: push
```

Comment:
I notice that for all the Personalisation repos, we just have a push trigger here, whereas several of the examples from the migration guide have explicit branch and pull request sections. Do you know what the difference is and is that something we should change?

Reply:
Sure, can do. I normally say on pushes to main, or pull requests, which covers the vast majority, but doesn't build if you're just saving a ton of commits on the regular before you're ready to PR them